packages/mdc-drawer/component.ts

```diff
@@ -68,6 +68,10 @@ export class MDCDrawer extends MDCComponent<MDCDismissibleDrawerFoundation> {
   private handleKeydown_!: SpecificEventListener<'keydown'>; // initialized in initialSyncWithDOM()
   private handleTransitionEnd_!: SpecificEventListener<'transitionend'>; // initialized in initialSyncWithDOM()
 
+  get list(): MDCList | undefined {
```

Comment:
May not be related to this PR. Looking at the implementation of drawer component, `list_` should not be `undefined`, drawer component should throw error when list is not found instead. Drawer foundation breaks at `focusActiveNavigationItem` adapter method if list is not found.

Reply:
I had wondered about this too, but we have the `list_` member currently declared as `MDCList?` and it looks like we're tolerating it being absent in `destroy`. So maybe there's indeed a separate bug to be had here...

Reply:
Upon further inspection/discussion, it looks like the adapter APIs also tolerate the list not existing, so in the interests of maintaining current behavior I'll keep this PR as-is.